openmetadata-ui/src/main/resources/ui/src/pages/CustomPropertiesPageV1/CustomPropertiesPageV1.tsx

```diff
@@ -216,15 +217,23 @@ const CustomEntityDetailV1 = () => {
               <ErrorPlaceHolder
                 buttons={
                   <Tooltip
-                    title={editPermission ? 'Add' : NO_PERMISSION_FOR_ACTION}>
-                    <ButtonAntd
+                    title={
+                      editPermission
+                        ? t('label.add-custom-entity-property', {
+                            entity: getCustomPageHeader().header,
+                          })
+                        : NO_PERMISSION_FOR_ACTION
```

Comment:
is it necessary to call `getCustomPageHeader()` function in tooltip? it will be called in every render!